apps/client/src/utils/index.js

```diff
@@ -90,3 +65,16 @@ export const scrollToRef = (ref, offset = 0) =>
  */
 export const stripString = (str) =>
   str && str.toLowerCase().replace(/[^a-z0-9]+/gi, "");
+
+/**
+ *
+ * This function return `true` or `false` wheter or not a value is a equal to a Dutch postalcode (eg: 1055XD)
+ *
+ * @param {string} value
+ */
+export const isValidPostalcode = (value) => {
+  const postalCodeRegex = /^[1-9][0-9]{3}[\s]?[A-Za-z]{2}$/i;
```

Comment:
I'm not really a regex guru. But the `/i` is not needed or you can replace `[A-Za-z]` with `[a-z]` I think.